### ISSUE TITLE
Page title is not meaningful - Patient Experience

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#3980]
## Area or Component
Patient Self-Registration and Patient Results

## Quick Summary
The title of the page is just "SimpleReport", which is a shared title for all pages in the patient experience.

## Accessibility Violation
Page <title> does not identify purpose of page
https://axe.deque.com/rule-help/en/title-not-meaningful

## Action Requested
Replace titles in the pages below with the following:

1. Terms of service: `Terms of service | SimpleReport`
2. Self-registration: `Test registration form | SimpleReport`
3. Self-registration complete: `Test registration complete | SimpleReport`
4. DOB input: `Test result verification | SimpleReport`
5. Patient results: `Test result | SimpleReport`

Spanish: 

1. Terms of service: `Condiciones del servicio | SimpleReport`
2. Self-registration: `Formulario de inscripción a la prueba | SimpleReport`
3. Self-registration complete: `Inscripción a la prueba completa | SimpleReport`
4. DOB input: `Verificación del resultado de la prueba | SimpleReport`
5. Patient results: `Resultado de la prueba | SimpleReport`

Pages:
1. The Terms of Service page (shared between self-registration and results)
<img width="702" alt="Screen Shot 2022-07-14 at 5 24 02 PM" src="https://user-images.githubusercontent.com/80282552/179122524-a2d99bf8-37fc-4e95-822a-8a2ee82d8b45.png">

2. The patient self-registration form page

<img width="771" alt="Screen Shot 2022-07-14 at 5 24 21 PM" src="https://user-images.githubusercontent.com/80282552/179122541-3feecc5c-7e07-460e-834c-355852e165e1.png">

3. The patient self-registration completion page
<img width="696" alt="Screen Shot 2022-07-14 at 5 23 04 PM" src="https://user-images.githubusercontent.com/80282552/179122468-756c309b-ba4b-45a6-9cd6-25d96c51da08.png">

4. The patient results DOB input
<img width="655" alt="Screen Shot 2022-07-14 at 5 24 52 PM" src="https://user-images.githubusercontent.com/80282552/179122588-566e4fd8-9ca1-410c-89bb-7bcb0fd7e3b8.png">

5. The patient results page
<img width="618" alt="Screen Shot 2022-07-14 at 5 25 20 PM" src="https://user-images.githubusercontent.com/80282552/179122629-79cd3283-9985-43f7-824d-83a2c20a2d8d.png">

## Source Code (optional)
Potentially the [PatientApp.tsx](https://github.com/CDCgov/prime-simplereport/blob/c3e6b2dc330021e425e01657eaee613c51f9e11c/frontend/src/patientApp/PatientApp.tsx#L1-L1)?

## Original Deque Issue
title-not-meaningful | The <title> element on this page does not identify the contents or purpose of the page. | Title is SimpleReport; should be SimpleReport \| Self-Registration or similar | Page <title> does not identify purpose of page | https://axe.deque.com/rule-help/en/title-not-meaningful | serious | Yes | No | Structure | ["title"] | The <title> element on this page does not identify the contents or purpose of the page. | <title>SimpleReport</title> | wcag2a,wcag242a | rnv7@cdc.gov | Patient Self Reg - Terms of Service | https://stg.simplereport.gov/app/register/9C7BB |   | 2022-07-13T21:08:07.783Z
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --

**Comment by lauraykerr** [comment]
Based on existing title pattern, recommend the following titles: 

1. Terms of service: `Terms of service | SimpleReport`
2. Self-registration: `Test registration form | SimpleReport`
3. Self-registration complete: `Test registration complete | SimpleReport`
4. DOB input: `Test result verification | SimpleReport`
5. Patient results: `Test result | SimpleReport`

**Comment by lauraykerr** [comment]
@zdeveloper Another patient experience ticket.

**Comment by lg-king** [comment]
Spanish translation needed

**Comment by lauraykerr** [comment]
Sent to translation team through Dana Pitts and Holly Wilson this morning (10/13)

**Comment by lauraykerr** [comment]
@lg-king @zdeveloper Spanish translations added to description!

**Comment by zdeveloper** [comment]
@lauraykerr  Awesome! thank you!
